packages/api/src/index.ts

```diff
@@ -63,6 +73,61 @@ const getApp = async () => {
 		}),
 	]);
 
+	apiRouter.get('/client-config', [
+		checkAuth,
+		asyncHandler(async (req, res) => {
+			const clientConfig: ClientConfig = {
+				googleClientId: config.auth.clientId,
+			};
+			res.send(JSON.stringify(clientConfig));
+		}),
+	]);
+
+	apiRouter.post('/export', [
+		checkAuth,
+		asyncHandler(async (req, res) => {
+			const exportRequest = TranscriptExportRequest.safeParse(req.body);
+			console.log(exportRequest);
+			const dynamoClient = getDynamoClient(
+				config.aws.region,
+				config.aws.localstackEndpoint,
+			);
+			if (exportRequest.success) {
+				const item = await getTranscriptionItem(
+					dynamoClient,
+					config.app.tableName,
+					exportRequest.data.id,
+				);
+				if (!item) {
+					const msg = `Failed to fetch item with id ${exportRequest.data.id} from database.`;
+					console.error(msg);
+					res.status(500).send(msg);
+					return;
+				}
+				const parsedItem = TranscriptionItem.safeParse(item);
+				if (parsedItem.success) {
+					const exportResult = await createTranscriptDocument(
+						config,
+						`${parsedItem.data.originalFilename} transcript`,
+						exportRequest.data.oAuthTokenResponse,
+						parsedItem.data.transcript.srt,
+					);
+					res.send({
+						documentId: exportResult,
+					});
```

Comment:
What if the `exportResult` is null? Do we not return error status in this case?

Reply:
Thanks, I've tidied this up